cmd/plugins/balloons/policy/balloons-policy.go

```diff
@@ -1202,6 +1209,12 @@ func (p *balloons) fillCloseToDevices(blnDefs []*BalloonDef) {
 		if blnDef.PreferIsolCpus {
 			blnDef.PreferCloseToDevices = append(blnDef.PreferCloseToDevices, virtDevIsolatedCpus)
 		}
+		if blnDef.CoreType == "performance" {
```

Comment:
It is fine to append both virtDevIsolatedCpus and either of virtDevPCores or virtDevECores to the blnDef.PreferCloseToDevices list?

Reply:
yes, because preference is given to isolated CPUs.